### PR TITLE
Error with settings  USE_TZ=True

### PR DESCRIPTION
**Pull request by zodman**
and tox error:

```
======================================================================
ERROR: test_set (tests.test_cli.CliTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/zodman/apps/django-constance/tests/test_cli.py", line 56, in test_set
    self.assertEqual(config.DATETIME_VALUE, datetime(2011, 9, 24, 12, 30, 25))
  File "/usr/lib/python2.7/unittest/case.py", line 513, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/usr/lib/python2.7/unittest/case.py", line 503, in _baseAssertEqual
    if not first == second:
TypeError: can't compare offset-naive and offset-aware datetimes

```

### CODE REVIEW

**Comment by zodman** [comment]
WIP

**Comment by zodman** [comment]
DONE!

well on CONSTANCE_CONFIG for a datetime, i see the datetime must be naive because you dont pass the timezone to de conf. So  i extend the compress method to convert the datetime aware to naive ever .. and some tweaks on unittest @jezdez  i waiting for you VoBo (LGFM)

File: tests/test_admin.py [hunk 1]
```diff
@@ -23,6 +28,42 @@ def setUp(self):
         self.normaluser.save()
         self.options = admin.site._registry[self.model]
 
+    # @mock.patch("constance.admin.ConstanceForm.clean_version", lambda x: 'test')
```
Comment (camilonova) [inline tests/test_admin.py]: Why this is commented?

File: tests/settings.py [hunk 2]
```diff
@@ -105,3 +105,10 @@
         },
     },
 ]
+
+TIME_ZONE = 'America/Merida'
+USE_I18N = True
+USE_L10N = True
+USE_TZ = True
+
```
Comment (camilonova) [inline tests/settings.py]: Leave a single blank line

File: tests/test_form.py [hunk 3]
```diff
@@ -24,3 +30,34 @@ def test_form_field_types(self):
...
+        f.save()
+        self.assertEqual(f.cleaned_data['DATETIME_VALUE'], now)
+
+    def tearDown(self):
+        initial_values = self.initial_values
+        values = (initial_values['DATETIME_VALUE'].date(),
+                  initial_values['DATETIME_VALUE'].time())
+        _set_constance_value('DATETIME_VALUE', values)
+
+
```
Comment (camilonova) [inline tests/test_form.py]: Leave a single blank line at the end of the file

File: tests/test_admin.py (hunk 1 above)
Comment (camilonova) [reply tests/test_admin.py]: We should not have any commented code, if is not needed then remove it.

Comment (zodman) [reply tests/test_admin.py]: remove it!

**Comment by camilonova** [comment]
Seems like there is a bunch of commented code, can you please look into it @zodman ?

**Pull request closed without merging**